src/main/app/claims/claimModelConverter.ts

```diff
@@ -37,13 +39,12 @@ export class ClaimModelConverter {
       claimData.interestDate = this.convertInterestDate(draftClaim.interestDate)
     }
     claimData.amount = new ClaimAmountBreakdown().deserialize(draftClaim.amount)
-    claimData.feeAmountInPennies = draftClaim.claimant.payment.amount
     claimData.claimants = [this.convertClaimantDetails(draftClaim)]
     claimData.defendants = [this.convertDefendantDetails(draftClaim)]
-    claimData.payment = draftClaim.claimant.payment
+    claimData.payment = this.convertPaymentDetails(draftClaim.claimant.payment)
```

Comment:
Another story will come in the future to simplify it and pass possibly only payment reference.